src/features/Home/components/ModelDetailsTable/index.tsx

```diff
@@ -0,0 +1,450 @@
+import React, { useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Column,
+  Row,
+  useFilters,
+  useGlobalFilter,
+  usePagination,
+  useSortBy,
+  useTable
+} from 'react-table';
+import { Button, Icon, Table as UswdsTable } from '@trussworks/react-uswds';
+import { ModelCategory } from 'gql/generated/graphql';
+import i18next from 'i18next';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import GlobalClientFilter from 'components/TableFilter';
+import TablePagination from 'components/TablePagination';
+import TableResults from 'components/TableResults';
+import TopScrollContainer from 'components/TopScrollContainer';
+import { formatDateLocal, formatDateUtc } from 'utils/date';
+import globalFilterCellText from 'utils/globalFilterCellText';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon,
+  sortColumnValues
+} from 'utils/tableSort';
+
+import { ModelPlansType } from '../ModelsByStatusGroup';
+
+import './index.scss';
+
+const ModelDetailsTable = ({
```

Comment:
This table is pretty much identical to the Home table.  That probably could have been refactored to use the same table.  It's a tad more work to extract the query from the home table, but I think it would be cleaner than recreating the same component.  On the fence about requiring this change...